.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,48 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+        default: ''
+
+jobs:
+  bump-version:
+    name: Bump Version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - uses: actions/cache@v2
+        name: Cache Gradle packages
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Set version
+        run: |
+          set -e
+          NEW_VERSION="${{ github.event.inputs.version }}"
+          sed -i "s/project\.externalVersion : '.*\..*\..*'/project\.externalVersion : '$NEW_VERSION'/g" build.gradle
+      - name: Print new version
+        run: |
+          echo "$(./gradlew -q currentVersion)"
+
+      - name: Setup git
+        run: |
+          set -ex
+          git config --global user.email "builder+skip-ci@codota.com"
+          git config --global user.name "codota-builder"
+
+      - name: Commit changes
+        run: |
+          NEW_VERSION="${{ github.event.inputs.version }}"
+          git pull origin master --no-edit
+          git commit -am "Set meta version to $NEW_VERSION"
+          git push
```

Comment:
you need the tag handling

Reply:
i mean you probably want to create and push the tag

Reply:
no, im leaving this to the production release action - it creates the tag and all.
i just want this to work as a first step, then we can later merge both actions -wdyt?

Reply:
the issue is that if i break the existing production release on master its gonna be a pain in my ass to fix, since I don't have permissions.
so each change will require a pr with merge approve and all that.
that's why I'm creating a separate action that doesn't change the existing one